### ISSUE TITLE
[MHV Portal / Accessibility] Newly presented error alert is not announced.

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#122600]
### 📝 User story

As a user who falls into scenario 1: "confirm email" of the Notification email update critical alerts work...
I want to select the "Confirm" button to indicate that my contact email is correct, 
so that I can dismiss the alert and continue using the website.

### 📝 Problem statement

When I select the button "Confirm" and the system can not confirm my email address, I am visually presented with an Error Alert. As a user who is using a screenreader (VO), this alert is not announced, and I am unaware that there was an error.

(?) Also, per [Figma](https://www.figma.com/design/CAChU51fWYMZsgDR5RXeSc/MHV-Landing-Page?node-id=7032-41911&t=TeQTGRqv9ZFSbw7l-1), there should be a close button on this error alert so that I can dismiss the error alert and return the page focus to the original alert.

(?) Focus Flow: selected the "Confirm" button and the error alert popsup, the focus remains on the "Confirm" button, it should really transfer to the new alert.

### 🌌 Supporting screenshots or videos

<img width="1053" height="458" alt="Image" src="https://github.com/user-attachments/assets/5972d943-e761-4e66-9298-be0b58de26d3" />

### 👣 Reproduction steps

From [test users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-mhv-cartographers.md#email-alerts-test-users) using test users for scenario 1: patient998 and using the VO screenreader
1. From the MyVA homepage or MHV homepage, when the confirm email alert is present
2. Select the "Confirm" button
3. You are correctly visually shown the Error Alert component, but this element is not announced via the screen reader.
4. The focus is not transfer to the Error Alert.

### 🔗 Point of contact

Steven Gallagher (@steveg-IAT)

### ⚠️ Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### 🔎 Relevant standards

**Accessibility Standard:**
- [SC 4.1.3 Status Messages](https://w3.org/TR/WCAG22#status-messages)  

**Experience Standard:**
- [VADS: USWDS: va-alert--with-aria-role](https://design.va.gov/storybook/?path=/story/uswds-va-alert--with-aria-role)
- [VADS: Assign appropriate ARIA role](https://design.va.gov/components/alert/#assign-an-appropriate-aria-role)

### 𝌣 Proposed solution or next steps

Following the VADS/USWDS...
https://design.va.gov/storybook/?path=/story/uswds-va-alert--with-aria-role

- Alert (Confirm): needs role="status"
- Alert (Error - we couldn't confirm...): needs role="alert"

### ✅ Acceptance criteria

- The newly added error alert is properly announced to the user
- (?) The newly added error alert can be dismissed

### ✅ Possible tasks

- [x] Review, refine, and acknowledge feedback.
- [x] Update ticket information (title, assignee, labels, project(s), etc.)
- [x] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by steveg-IAT** [comment]
**:exclamation:NOTE** 
The new alert focus highlight is different between the MHV landing and the profile page... Profile has specific scss to exclude the outline on [tabindex="-1"] selector. These should be sync'd up.

**Comment by sarahhortonadhoc** [comment]
Still having issues with access and reproducing the scenario. None of the test users from [Scenario 1](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-mhv-cartographers.md#scenario-1-should-see-critical-action-alerts-and-confirm-contact-email-on-profile) is displaying the confirm email alert.

**Comment by carlosfelix2** [comment]
It is not going to be possible to generate a 500 error in staging to see this alert. I recommend running this locally with local mocks to generate the alert.

**Comment by sarahhortonadhoc** [comment]
See [Slack thread on QA/Accessibility Review](https://dsva.slack.com/archives/C096YQLRWHK/p1764864461531689).
